added edit (as per post) and support for osx

encodes1 opened the pull request:
Updated to update a specific line
also added support for osx.

Commit:
added update function and added osx suppurt

jasonvanzin commented:
Someone on reddit made a good observation. Instead of adding osx, basically all you need to do is check for Windows. If it's not Windows, it's /etc/hosts.

Commit:
updated the testing on environment as per redis post, only check for windows

Commit:
updating finding of position and removing the extra new lines

encodes1 commented:
Added a few tweaks :)

Commit:
updated readme and fixed typo in usage

encodes1 commented:
Added a better readme. And fixed a typo in the code.
Now uses the win32 or /etc/hosts
updated the where in file as there was some issues with my previous code.
Enjoy

encodes1 commented:
Are you accepting the pull request?
